data/json/npcs/isherwood_farm/NPC_Chris_Isherwood.json

```diff
@@ -17,7 +17,12 @@
     "id": "NC_ISHERWOOD_CHRIS",
     "name": { "str": "Chris Isherwood" },
     "job_description": "I'm Jesse and Carlos' son, part of the Isherwood family",
-    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "traits": [
+      { "group": "NPC_starting_traits" },
+      { "group": "Appearance_demographics" },
+      { "trait": "NO_BASH" },
+      { "trait": "RETURN_TO_START_POS" }
```

Comment:
Same here. In Chris' mission you need to take him to a Mi-Go tower and this prevents that